web/urls.py

```diff
@@ -10,6 +15,20 @@
 from web.datasets.admin import public_admin
 
 
+
+schema_view = get_schema_view(
+   openapi.Info(
+      title="Maria Quiteria API",
+      default_version='v1',
+      description="",
+      terms_of_service="https://www.google.com/policies/terms/",
+      contact=openapi.Contact(email="contact@snippets.local"),
+      license=openapi.License(name="BSD License"),
```

Comment:
```suggestion
      license=openapi.License(name="MIT"),
```

É bom mantermos a mesma licença dos nossos repositórios.